sigstore/sign.py

```diff
@@ -324,29 +324,7 @@ def __init__(
         self._tsa_clients = tsa_clients or []
 
     @classmethod
-    def production(cls) -> SigningContext:
-        """
-        Return a `SigningContext` instance configured against Sigstore's production-level services.
-        """
-        return cls(
-            fulcio=FulcioClient.production(),
-            rekor=RekorClient.production(),
-            trusted_root=TrustedRoot.production(),
-        )
-
-    @classmethod
-    def staging(cls) -> SigningContext:
-        """
-        Return a `SignerContext` instance configured against Sigstore's staging-level services.
-        """
-        return cls(
-            fulcio=FulcioClient.staging(),
-            rekor=RekorClient.staging(),
-            trusted_root=TrustedRoot.staging(),
-        )
```

Comment:
Same note about public API + breaking changes as above!

Reply:
yeah, I should probably hide the similar methods in RekorClient now as well (they are still used by the `fix-bundle` special case but I could make them more private)